plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlClient.java

```diff
@@ -276,6 +290,23 @@ public void testConvertLike()
                 .hasValue("(\"c_varchar\") LIKE ('%pattern\\%') ESCAPE ('\\')");
     }
 
+    @Test
+    public void testConvertIn()
+    {
+        assertThat(JDBC_CLIENT.convertPredicate(
+                TestingConnectorSession
+                        .builder()
+                        .setPropertyMetadata(new JdbcMetadataSessionProperties(new JdbcMetadataConfig().setDomainCompactionThreshold(10), Optional.empty()).getSessionProperties())
+                        .build(),
```

Comment:
1. why non-default compaction threshold?
2. with defaults, it should be defined as SESSION constant on the class level. Other test methods don't need it today, but they may in the future, and this is just a better testing SESSION to be used here,